test/src/test/java/hudson/model/ComputerTest.java

```diff
@@ -263,4 +266,21 @@ public void testTerminatedNodeAjaxExecutorsDoesNotShowTrace() throws Exception {
         j.assertBuildStatus(Result.FAILURE, j.waitForCompletion(b));
     }
 
+    @Test
+    public void computersCollected() throws Exception {
```

Comment:
may not pass on windows `ComputerTest` failed in https://ci.jenkins.io/blue/organizations/jenkins/Core%2Fjenkins/detail/PR-8640/3/pipeline/235

Reply:
Hmm, though `junit` did not catch that for some reason.